mail-client/alpine/alpine-2.21.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils flag-o-matic autotools toolchain-funcs
```

Comment:
Are ``eutils`` and ``flag-o-matic`` needed anymore?

Reply:
Removed.